### PR TITLE
New blog post on new releases

### PR DESCRIPTION
**Pull request by michalfabik**
See commit  

Signed-off-by: Michal Fabik <mfabik@redhat.com>

### CODE REVIEW

File: _posts/2020-02-19-new-releases.markdown [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
++ Add the possibility to run retraces in a podman container
++ Rework the pre- and post-retrace hook mechanism
+
+--- 
+
+More detailed changelogs are available for [abrt](https://github.com/abrt/abrt/blob/2.14.0/CHANGELOG.md) and [libreport](https://github.com/abrt/libreport/blob/2.12.0/CHANGELOG.md).
+
+
+[gnome-abrt-GUI]: /assets/2020-02-new-releases/gnome-abrt-GUI.png "Updated GUI example"
+[gnome-abrt]: raw:gnome&#8209;abrt
```
Comment (mgrabovsky) [inline _posts/2020-02-19-new-releases.markdown]: What is this line supposed to do?

Comment (michalfabik) [reply _posts/2020-02-19-new-releases.markdown]: Nothing any more, removed, thanks.

**Comment by mgrabovsky** [comment]
Looks great.

**Comment by mkutlak** [comment]
IMO, I would only write about noticeable changes or "major" ones.

For example I wouldn't mention these.
```
abrt-java-connector
Fix build failure with GCC 10

gnome-abrt
Change build system to Meson 
```
---

```
CLI fixes 
```
What was actually fixed?

---

```
GUI Tweaks
```
What tweaks?

---

```
dbus: Warn the user when GetProblems() is called with a large (>100) number of problems 
```

```
abrt-dbus will now warn the user about a large number of problems on the machine
```
 
---

```
Rework the pre- and post-retrace hook mechanism 
```

```
The hook mechanism was reworked ... for mor see docs - https://github.com/abrt/retrace-server/wiki/Hook-scripts
```

So on, so on, see https://github.com/abrt/abrt/blob/2.12.1/CHANGELOG.md#2100 for "inspiration".

**Comment by mkutlak** [comment]
Oh, too late.

**Comment by ernestask** [comment]
Changing the build system is quite major. If there’s any entity out there that ships anything ABRT and isn’t Fedora, announcing that would prevent surprises (see: every instance of that happening).

**Pull request merged**